experimenter/experimenter/nimbus-ui/src/components/Summary/index.tsx

```diff
@@ -16,6 +16,8 @@ import RequestLiveUpdate from "src/components/Summary/RequestLiveUpdate";
 import TableAudience from "src/components/Summary/TableAudience";
 import TableBranches from "src/components/Summary/TableBranches";
 import TableOverview from "src/components/Summary/TableOverview";
+import TableQA from "src/components/Summary/TableQA";
```

Comment:
QATable?

Reply:
Ugh, I know. It sounds so much better to be `QATable`. All of the others are `TableX`, so this follows the naming convention. I think we might want to do a little refactoring in this Summary area (there's also Summary **and** PageSummary). I think I'll file something for that...

Reply:
:+1: for consistency and later refactor